src/Service/Sqs/SQSMessengerTransportConfigurationInterface.php

```diff
@@ -0,0 +1,11 @@
+<?php
+
+namespace Bref\Symfony\Messenger\Service\Sqs;
+
+use InvalidArgumentException;
+
+interface SQSMessengerTransportConfigurationInterface
```

Comment:
Is there a reason to introduce a new interface?

If there's no specific reason I'd rather keep the API surface as small as possible to facilitate maintenance